Documentation/contributing/docs/docsframework.rst

```diff
@@ -0,0 +1,382 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _docs_framework:
+
+***********************
+Documentation framework
+***********************
+
+This page contains notes on the framework in use for Cilium documentation. Its
+objective is to help contributors understand the tools and build process for
+the documentation, and to help maintain it.
+
+Alas, this is the sort of document that can quickly get out of date. Please
+double-check the information with the codebase for accuracy when you need it,
+and update this page if you find discrepancies.
+
+Sphinx
+======
+
+Cilium relies on Sphinx to generate its documentation.
+
+Sphinx features
+---------------
+
+Here are some specific Sphinx features in use in Cilium's documentation:
+
+- `Tab groups`_
+
+- `OpenAPI`_ documentation generation
+
+- Mark-up languages: reStructuredText (rST) and Markdown (`MyST`_ flavor)
+
+- Substitutions, for example:
+
+  - ``|SCM_WEB|``
+  - ``|CHART_VERSION|``
+
+- Multiple versions (for all supported branches, plus two aliases: ``stable``
+  and ``latest``)
+
+.. _OpenAPI: https://github.com/sphinx-contrib/openapi
+.. _Tab groups: https://github.com/executablebooks/sphinx-tabs/
+.. _MyST: https://myst-parser.readthedocs.io
+
+Sphinx version
+--------------
+
+The version of Sphinx in use is defined in
+``Documentation/requirements-min/requirements.txt``. See the :ref:`section on
+requirements <docs_requirements>` for more details.
+
+Auto-generated contents
+=======================
+
+Some contents are automatically generated at build time. The Documentation
```

Comment:
I think we can use lower case, no?
```suggestion
Some contents are automatically generated at build time. The documentation
```

Reply:
Yes, this is a typo. I wrote this doc as a list of brief notes at first, I expanded them today but forgot to remove the capital letter after adding the article.